services/adgs/config/adgs_ws_config.template.yaml

```diff
@@ -107,9 +107,36 @@ template:
             - "Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'platformShortName' and att/OData.CSC.StringAttribute/Value eq '{attr_platform_short_name}')"
             - "Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'processingDate' and att/OData.CSC.StringAttribute/Value eq '{attr_processing_date#to_iso_utc_datetime}')"
 
+    sort:
+      sort_by_tpl: '&$orderby={sort_param} {sort_order}'
+      sort_param_mapping:
+        id: Name
+        auxip:id: Id
+        created: PublicationDate
+        #datetime: Start # How to access ContentDate['Start']?
+        #start_datetime: Start
+        end_datetime: End
+        file:size: ContentLength
+        type: ContentType
+        eviction_datetime: EvictionDate
+        #product:type: productType
+        #processing:datetime: processingDate
+        published: PublicationDate
+        #file:checksum: ChecksumValue
+        #platform: platformSerialIdentifier
+        #constellation: platformShortName
+        #processing:facility: processingCenter
+        startTimeFromAscendingNode: ContentDate.Start
+        completionTimeFromAscendingNode: ContentDate.Stop
```

Comment:
I made a mistake it's a slash: ContentDate/Stop. I do the remark only once but there are more occurrences to fix